dune

```diff
@@ -25,15 +21,12 @@
   (js_of_ocaml
    (compilation_mode separate)
    (flags
-    (:standard --enable effects --effects double-translation))
+    (:standard --effects double-translation))
    (build_runtime_flags
-    (:standard --enable effects --effects double-translation)))
+    (:standard --effects double-translation)))
```

Comment:
You should grep for `build_runtime_flags` and remove this option wherever it is not needed.